packages/components/src/range-control/index.tsx

```diff
@@ -41,6 +41,15 @@ import { space } from '../utils/space';
 
 const noop = () => {};
 
+function useUniqueId( idProp?: string ) {
+	const id = useInstanceId(
+		UnforwardedRangeControl,
+		'inspector-range-control'
```

Comment:
I think that's the instance ID prefix that many components in the package adopt 😓  Not sure how easy it would be to refactor away from it, but probably not in the scope for this PR (`'inspector-range-control'` is already used in `trunk` and in this PR it was moved to the `useUniqueId` function)